base/uk.ac.stfc.isis.ibex.opis/resources/Razorbill_RP100.opi

```diff
@@ -0,0 +1,1306 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
OPI doesn't really follow the style conventions...

How about two grouping containers (one for each channel), each of which have the controls for a single channel in our standard scheme inside it?

Can discuss if you prefer...